core/store/src/db.rs

```diff
@@ -142,6 +142,19 @@ fn col_name(col: DBCol) -> String {
     format!("col{}", col as usize)
 }
 
+fn ensure_max_open_files_limit(max_open_files: i32) -> () {
+    // We’re configuring each RocksDB to use max_open_files file descriptors. On top of that we can
+    // have some other file descriptors opened by neard process so we use the value of
+    // 2 * max_open_files to be sure that the binary can correctly run.
+    let (soft, hard) = rlimit::Resource::NOFILE.get().unwrap();
+    let required = max_open_files as u64 + 1025;
+    if soft < required {
+        assert!(hard >= required, concat!("Can't run near binary since hard limit for the number ",
+                "of opened files is too small: {} required: {}"), hard, 2 * max_open_files);
```

Comment:
There's no need for concat just use backslash, e.g.:

```rust
        assert!(hard >= required,
                "Can’t run neard: hard limit for the number of opened files is \
                 too small: {}, required: {}", hard, required);
```

Note space before backslash.  In Rust backslash before a newline in a string
eats up all following white space.

Also, this needs to use `required` rather than `2 * max_open_files`.